### PR TITLE
chore: make standards cards 3 column layout + full card clickable

### PR DESCRIPTION
**Pull request by CJ42**
Adjust the style and large padding of the cards + make the link clickable for the full card (whole `<div>`), not only the text.

## After

image

## Before

image

### CODE REVIEW

File: src/components/StandardsGallery/index.tsx [hunk 1]
```diff
@@ -133,42 +133,44 @@ const STANDARDS = [
...
-    </div>
+    <Link
+      className={`col col--4 margin-bottom--lg ${styles.legoCard}`}
+      to={link}
+      style={{
+        borderWidth: '1px',
+      }}
+    >
+      <h2 className="flex items-center font-jakarta">
+        <div>{title}</div>
```
Comment (Hugoo) [inline src/components/StandardsGallery/index.tsx:144]: We usually avoid to put block elements like this div inside a heading. You probably want to use a span or something inline.

Check

https://stackoverflow.com/questions/66084722/does-html5-allow-a-block-level-element-like-div-inside-a-heading-like-h1

File: src/components/StandardsGallery/index.tsx [hunk 2]
```diff
@@ -133,42 +133,44 @@ const STANDARDS = [
...
+      </h2>
+      <h3 className="flex items-center font-jakarta">
+        <div className={styles.standardName}>{name}</div>
```
Comment (Hugoo) [inline src/components/StandardsGallery/index.tsx:147]: Can you replace this div with a span?

**Review by Hugoo — CHANGES_REQUESTED** [review]

File: src/components/StandardsGallery/index.tsx (hunk 1 above)
Comment (Hugoo) [reply src/components/StandardsGallery/index.tsx:144]: Here I think this div is useless anyway.

Comment (CJ42) [reply src/components/StandardsGallery/index.tsx:144]: true I don't know why it's there. Probably a mistake

**Review by Hugoo — APPROVED** [review]

**Pull request merged**